gateway/Dockerfile

```diff
@@ -29,6 +29,7 @@ FROM chef AS testing_env
 RUN rustup component add llvm-tools && \
     rustup component add rustfmt && \
     cargo install grcov
+RUN cargo install --locked --git https://github.com/tbrand/cargo-tomlfmt --rev 85354dfd771be75d7290435e613c07cb77b26e96
```

Comment:
I don't think that's the right place to perform the check. I'd rather have it done directly in a github action, just like here: https://github.com/servo/servo/pull/30128/files

Reply:
OK, so the link you provided above is actually doing `cargo install`. Although it's cache so it should not be a CI timing issue, I'd rather keep all the tools installed the same way, inside the Docker image of each application (like `rustfmt`, `grcov` or `diesel` are for example).

Reply:
In the end, still used the Github Actions `cargo-install` since I couldn't install taplo in docker images: taplo is ran on the entire repository.